v2.0/admin-ui-node-map.md

```diff
@@ -0,0 +1,185 @@
+--store=node1 \
+--host=localhost \
+--port=26257 \
+--http-port=8080 \
+--join=localhost:26257,localhost:26258,localhost:26259
+~~~
+
+In a new terminal, start Node 2:
+
+{% include copy-clipboard.html %}
+~~~ shell
+./cockroach start \
+--insecure \
+--locality=region=us-east,datacenter=us-east-1 \
+--store=node2 \
+--host=localhost \
+--port=26258 \
+--http-port=8081 \
+--join=localhost:26257,localhost:26258,localhost:26259
+~~~
+
+In a new terminal, start Node 3:
+
+{% include copy-clipboard.html %}
+~~~ shell
+./cockroach start \
+--insecure \
+--locality=region=us-west,datacenter=us-west-1 \
+--store=node3 \
+--host=localhost \
+--port=26259 \
+--http-port=8082 \
+--join=localhost:26257,localhost:26258,localhost:26259
+~~~
+
+In a new terminal, start Node 4:
+
+{% include copy-clipboard.html %}
+~~~ shell
+./cockroach start \
+--insecure \
+--locality=region=eu-west,datacenter=eu-west-1 \
+--store=node4 \
+--host=localhost \
+--port=26260 \
+--http-port=8083 \
+--join=localhost:26257,localhost:26258,localhost:26259
+~~~
+
+In a new terminal, use the [`cockroach init`](initialize-a-cluster.html) command to perform a one-time initialization of the cluster:
+
+{% include copy-clipboard.html %}
+~~~ shell
+./cockroach init --insecure
+~~~
+
+#### Step 3. [Set the enterprise license](enterprise-licensing.html)
+
+#### Step 4. Insert the latitudes and longitudes of each region into the `systems.locations` table
+
+{% include copy-clipboard.html %}
+~~~ sql
+> INSERT INTO system.locations VALUES
+  ('region', 'us-east', 40.367474, -82.996216), 
+  ('region', 'us-west', 43.8041334, -120.55420119999997), 
+  ('region', 'eu-west', 48.856614, 2.3522219000000177);
+~~~
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> won't be displayed until all regions are assigned the corresponding latitudes and longitudes. {{site.data.alerts.end}}
+
+To get the latitudes and longitudes of AWS regions, see [Locations Co-ordinates for Reference](#location-co-ordinates-for-reference).
+
+#### Step 5. Verify the settings 
+
+Navigate to `https://localhost:8080/#/reports/localities` and use the **Localities** table to verify that the latitudes and longitudes correspond to the correct regions.
+
+#### Step 6. View the Node Map
+
+[Navigate to the **Overview page**](admin-ui-cluster-overview.html) and switch to the **Node Map** view from the drop-down box.
+
+#### Step 7. Navigate the Node Map
+
+Suppose you want to navigate to Node 2, which is in datacenter `us-east-1` in the region `us-east`. To navigate to Node 2:
+
+1. Click on map component marked as **region=us-east** on the Node Map. The datacenter view is displayed.
+2. Click on the datacenter component marked as **datacenter=us-east-1**. The individual node components are displayed.
+3. To navigate back to the cluster view, either click on **Cluster** in the bread-crumb trail at the top of the Node Map, or click **Up to Cluster** in the lower left-hand side of the Node Map.
```

Comment:
If they're two levels deep, the up button will say "Up to region=us-east".

Reply:
The "up" button only goes up one level.  If they are down two levels (one from step 1, and a second on step 2), the button will only go up one level, to **region=us-east**.